src/project/qml/MuseScore/Project/internal/Export/ExportOptionsView.qml

```diff
@@ -76,14 +83,18 @@ ColumnLayout {
     }
 
     ExportOptionItem {
+        width: parent.width
         visible: subtypeComboBox.count > 0
         text: qsTrc("project", "File type:")
-        firstColumnWidth: privateProperties.firstColumnWidth
 
         Dropdown {
             id: subtypeComboBox
             Layout.fillWidth: true
 
+            navigation.name: "ExportSubtypeDropdown"
+            navigation.panel: navPanel
+            navigation.row: 2
```

Comment:
you need to control the navigation(enable state) depending on the visibility of the parent
If the parent is invisible, then the navigation will still work, because it is just an object

navigation.enable: option.visible

Reply:
I think this is not necessary: this is already being done in DropdownItem.qml (which is used as the base for the Dropdown component).
And in my experience, when the parent QQuickItem is invisible, the child items will be invisible too.

Also, when I test it on my computer, it works fine.

Reply:
All right. I described another case